### ISSUE TITLE
TF201077: The work item type Product Backlog Item cannot be found.

### ISSUE THREAD

**Issue by eeerrrwwwuuu** [solidify/jira-azuredevops-migrator#37]
I'm having some problem with Importing tickets from JIRA due to problem with mapping types.

```
  "type-map":{
    "type": [
      {
        "source": "Feature",
        "target": "Feature"
      },
 .....
      {
        "source": "Story",
        "target": "Product Backlog Item"
.....
      {
        "source": "Task",
        "target": "Product Backlog Item"
      }

```

ech of tickets stored in Json starts with : 
```
{"Type":"Product Backlog Item", "OriginId":"ITSWBIOE-721","WiId":-1, 
```
what leads to an error  
```
[Microsoft.TeamFoundation.WorkItemTracking.Client.WorkItemTypeDeniedOrNotExistException] TF201077: The work item type Product Backlog Item cannot be found. It may have been renamed or destroyed.:
   w Microsoft.TeamFoundation.WorkItemTracking.Client.WorkItemTypeCollection.get_Item(String typeName)
   w WorkItemImport.Agent.CreateWI(String type)
   w WorkItemImport.ImportCommandLine.ExecuteMigration(CommandOption token, CommandOption url, CommandOption configFile, Boolean forceFresh)
```

If I understand it correctly, there's no match in target schema for this type, I believe I should remap it somehow ? I was thinking about : 

```
     {
        "source": "Story",
        "target": "User Story"
      },
```       
and 
```
      {
        "source": "Task",
        "target": "Task"
      },
```

**Comment by MOlausson** [comment]
Hi @eeerrrwwwuuu, you are right that if the work item types don't exist in your target azure devops project (like Product Backlog Item in your case) it will err out and agree that the error could be better. I'll see if we can improve that.

But the solution is as you describe to map to the correct types (User Story in your case).

**Comment by MOlausson** [comment]
Fixed in release 2.2.x
